app/src/main/java/com/monkeyteam/chimpagne/ui/components/DateSelector.kt

```diff
@@ -77,7 +77,7 @@ fun DateSelector(
       text =
           DateFormat.getDateInstance(DateFormat.MEDIUM).format(selectedDate.time) +
               if (selectTimeOfDay) {
-                " at " + DateFormat.getTimeInstance(DateFormat.SHORT).format(selectedDate.time)
+                " @ " + DateFormat.getTimeInstance(DateFormat.SHORT).format(selectedDate.time)
```

Comment:
I actually preferred "at" instead of "@"

Reply:
For the second reviewer : my point is to use "@" for both english and english instead of "at"